### ISSUE TITLE
kgctl graph command requires user to provide mesh-granularity

### ISSUE THREAD

**Issue by stv0g** [squat/kilo#216]
If I run `kgctl graph` get the following error message:

```
Error: failed to obtain granularity: mesh granularity auto is not supported
```

I have configured my kg DaemonSet to use a full mesh.
I was wondering, why do we need to specify the mesh granularity by hand. 
Its not a nice UX.

**Comment by leonnicolas** [comment]
This was solved in #197 
Are you using a late enough version?

**Comment by stv0g** [comment]
Oh thanks. Yes, I've been using the latest release.
